### ISSUE TITLE
flow-cli detects crash on windows

### ISSUE THREAD

**Issue by spacepluk** [onflow/flow-cli#612]
I'm getting "Crash Detected!" on windows when running these:

```
flow deploy --update
flow dev-wallet

<nil>
[] []
map[device:map[arch:amd64 num_cpu:8] os:map[name:windows] runtime:map[go_maxprocs:8 go_numcgocalls:1190 go_numroutines:4 name:go version:go1.18.5]]
goroutine 1 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x65
github.com/onflow/flow-cli/internal/command.initCrashReporting.func1(0xc0005ca000, 0xc00037bbc0)
        github.com/onflow/flow-cli/internal/command/command.go:313 +0x22d
github.com/getsentry/sentry-go.(*Client).processEvent(0xc0005b2420, 0xc0005ca000, 0xc00037bbc0, {0x2014040, 0xc0004d0000})
        github.com/getsentry/sentry-go@v0.13.0/client.go:548 +0x20e
github.com/getsentry/sentry-go.(*Client).CaptureEvent(...)
        github.com/getsentry/sentry-go@v0.13.0/client.go:365
github.com/getsentry/sentry-go.(*Client).CaptureException(0xc000513600?, {0x20139a0?, 0xc0006061d0?}, 0xc0006061d0?, {0x2014040, 0xc0004d0000})
        github.com/getsentry/sentry-go@v0.13.0/client.go:356 +0x65
github.com/getsentry/sentry-go.(*Hub).CaptureException(0xc00043b8f0, {0x20139a0?, 0xc0006061d0})
        github.com/getsentry/sentry-go@v0.13.0/hub.go:260 +0xbe
github.com/getsentry/sentry-go.CaptureException(...)
        github.com/getsentry/sentry-go@v0.13.0/sentry.go:48
github.com/onflow/flow-cli/internal/command.Command.handleUserTracking({0x2b9caa0, {0x17f9440, 0x2bae830}, 0x0, 0x1e4dd70})
        github.com/onflow/flow-cli/internal/command/command.go:87 +0x6a
github.com/onflow/flow-cli/internal/command.Command.AddToParent.func1(0x2b9caa0?, {0x2c11108, 0x0, 0x0})
        github.com/onflow/flow-cli/internal/command/command.go:110 +0xd3
github.com/spf13/cobra.(*Command).execute(0x2b9caa0, {0xc0000de070, 0x0, 0x1})
        github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xc000125b80)
        github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        github.com/onflow/flow-cli/cmd/flow/main.go:82 +0x4f5
```

It continues the execution but it interrupts my dev scripts.

**Comment by sukantoraymond** [comment]
will look into this, thanks! @spacepluk

**Comment by sukantoraymond** [comment]
Tracking can be disabled by running `flow config metrics disable`, thanks!

**Comment by spacepluk** [comment]
thanks! I tried to disable it but it doesn't seem like that's working on windows
![Screen Shot 2022-08-12 at 18 40 19](https://user-images.githubusercontent.com/824762/184407981-bf45756a-ee4f-438c-887a-b5dba2d512a6.png)
:

**Comment by sideninja** [comment]
@spacepluk Hey! Thank you for reporting this to us. It looks like there is a bug in the CLI on Windows. Can you share with us what version of the CLI you have? Is this the first time you see this happening (it worked before)? 
Also can you share what you see in the output if you click on "No" when you get the report a crash prompt.

**Comment by sukantoraymond** [comment]
copied over from discord
````
<nil>
[] []
map[device:map[arch:amd64 num_cpu:8] os:map[name:windows] runtime:map[go_maxprocs:8 go_numcgocalls:1190 go_numroutines:4 name:go version:go1.18.5]]
goroutine 1 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x65
github.com/onflow/flow-cli/internal/command.initCrashReporting.func1(0xc0005ca000, 0xc00037bbc0)
        github.com/onflow/flow-cli/internal/command/command.go:313 +0x22d
github.com/getsentry/sentry-go.(*Client).processEvent(0xc0005b2420, 0xc0005ca000, 0xc00037bbc0, {0x2014040, 0xc0004d0000})
        github.com/getsentry/sentry-go@v0.13.0/client.go:548 +0x20e
github.com/getsentry/sentry-go.(*Client).CaptureEvent(...)
        github.com/getsentry/sentry-go@v0.13.0/client.go:365
github.com/getsentry/sentry-go.(*Client).CaptureException(0xc000513600?, {0x20139a0?, 0xc0006061d0?}, 0xc0006061d0?, {0x2014040, 0xc0004d0000})
        github.com/getsentry/sentry-go@v0.13.0/client.go:356 +0x65
github.com/getsentry/sentry-go.(*Hub).CaptureException(0xc00043b8f0, {0x20139a0?, 0xc0006061d0})
        github.com/getsentry/sentry-go@v0.13.0/hub.go:260 +0xbe
github.com/getsentry/sentry-go.CaptureException(...)
        github.com/getsentry/sentry-go@v0.13.0/sentry.go:48
github.com/onflow/flow-cli/internal/command.Command.handleUserTracking({0x2b9caa0, {0x17f9440, 0x2bae830}, 0x0, 0x1e4dd70})
        github.com/onflow/flow-cli/internal/command/command.go:87 +0x6a
github.com/onflow/flow-cli/internal/command.Command.AddToParent.func1(0x2b9caa0?, {0x2c11108, 0x0, 0x0})
        github.com/onflow/flow-cli/internal/command/command.go:110 +0xd3
github.com/spf13/cobra.(*Command).execute(0x2b9caa0, {0xc0000de070, 0x0, 0x1})
        github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xc000125b80)
        github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        github.com/onflow/flow-cli/cmd/flow/main.go:82 +0x4f5
````

**Comment by sukantoraymond** [comment]
Issue solved with release v0.39.0
